### PR TITLE
Added youtube playlist link for android development

### LINKED ISSUE #1104: [Add] Add youtube playlist link for android development
### Site URL

https://www.youtube.com/@CodeWithHarry/playlists

### Description of the above provided site

This android tutorials in Hindi explains how android applications are built using Android studio. In this Android Java Course one can learn how android applications are built using programming in java and XML. This android course will teach you everything you need to learn to create your first Android Application!

### Category

Youtube

### Sub Category

Youtube

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by madhurafulkar**
##1104
Closes #1104 

## Changes proposed

Added youtube playlist link for android development

## Screenshots
Screenshot 2023-06-25 180723

### CODE REVIEW

File: database/youtube/android.json [hunk 1]
```diff
@@ -30,5 +30,13 @@
     "category": "youtube",
     "subcategory": "android",
     "language": "hindi"
+  },
+  {
+    "name": "Android Development Tutorials in Hindi | CodeWithHarry",
+    "description": "This android tutorials in Hindi explains how android applications are built using Android studio. In this Android Java Course one can learn how android applications are built using programming in java and XML.",
```
Comment (CBID2) [inline database/youtube/android.json:36]: [Typo]
```suggestion
    "description": "In this Android Java Course, one can learn how Android applications are built using programming in Java and XML.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @madhurafulkar! 😄 Aside from the proofreading error in my in-line comment, you are off to a great start.

**Review by CBID2 — APPROVED** [review]
LGTM

**Review by ujjawaltyagii — APPROVED** [review]
lgtm

**Pull request merged**